### DISCUSSION TITLE
Auto-backups?

### DISCUSSION THREAD

**Discussion by Cuel** [mbround18/valheim-docker#96] [Ideas]
I don't see anything about automatically backing up saves?  With the world eater bug it seems like a good idea

**Comment by mbround18** [comment]
> I don't see anything about automatically backing up saves? With the world eater bug it seems like a good idea

What I can do to simplify the process is look into adding a backup command to `Odin` which would zip the world files <3 

But it will be up to you to find where to place the backup files :P 

For example, I am using Backblaze B2 for my backup solution and using [istepanov/backup-to-s3](https://hub.docker.com/r/istepanov/backup-to-s3/) to execute that task. 

Another tool I would recommend is [loomchild/volume-backup](https://hub.docker.com/p/loomchild/volume-backup) as it can backup volumes for docker. 

Additionally, another tool I would like to recommend is [eeacms/rsync](https://hub.docker.com/r/eeacms/rsync/).

**Comment by mbround18** [comment]
@Cuel I should have a backup solution up a little later <3

**Reply by kpengwin** [reply]
Thanks! I'll be using this as well, was working on a bash script today to grab backups and put them somewhere safe.

**Comment by mbround18** [comment]
Auto backup feature: https://github.com/mbround18/valheim-docker/pull/100

**Reply by Cuel** [reply]
Great!

**Comment by mbround18** [comment]
This feature has been completed and tested on my servers. Its working like a charm.

**Reply by kpengwin** [reply]
Also working perfectly for me for several days
